src/main/java/org/breedinginsight/brapps/importer/controllers/UploadController.java

```diff
@@ -114,7 +114,7 @@ public HttpResponse<Response<ImportResponse>> commitData(@PathVariable UUID prog
                                                              @PathVariable UUID uploadId, @Body @Nullable Map<String, Object> userInput) {
         try {
             AuthenticatedUser actingUser = securityService.getUser();
-            ImportResponse result = fileImportService.updateUpload(programId, uploadId, actingUser, userInput, true);
+            ImportResponse result = fileImportService.updateUpload(programId, uploadId, null, actingUser, userInput, true);
```

Comment:
That is, what is the desired behavior?

Reply:
This endpoint doesn't have the workflow id path parameter so it would be for non-experiment import types that don't have workflows like germplasm. No workflow information is required but a shared interface between workflow and non-workflow imports is used which is why that is set to null.